README.md

```diff
@@ -1,147 +1,68 @@
-
-### Manual installation
-
-#### iOS
-
-1. In XCode, in the project navigator, right click `Libraries` ➜ `Add Files to [your project's name]`
-2. Go to `node_modules` ➜ `react-native-zoom-us` and add `RNZoomUs.xcodeproj`
-3. In XCode, in the project navigator, select your project. Add `libRNZoomUs.a` to your project's `Build Phases` ➜ `Link Binary With Libraries`
-4. Run your project (`Cmd+R`)<
-5. Follow [Mostly automatic installation-> Extra steps for iOS](#extra-steps-for-ios)
-
-#### Android
-
-1. Open up `android/app/src/main/java/[...]/MainActivity.java`
-  - Add `import ch.milosz.reactnative.RNZoomUsPackage;` to the imports at the top of the file
-  - Add `new RNZoomUsPackage()` to the list returned by the `getPackages()` method
-2. Append the following lines to `android/settings.gradle`:
-  	```
-  	include ':react-native-zoom-us'
-  	project(':react-native-zoom-us').projectDir = new File(rootProject.projectDir, 	'../node_modules/react-native-zoom-us/android')
-  	```
-3. Insert the following lines inside the dependencies block in `android/app/build.gradle`:
-  	```
-      compile project(':react-native-zoom-us')
-  	```
-4. Follow [Mostly automatic installation-> Extra steps for Android](#extra-steps-for-android)
-
-
 ## Usage
-```ts
+```typescript
 import ZoomUs from 'react-native-zoom-us';
 
-await ZoomUs.initialize(
-  config.zoom.appKey,
-  config.zoom.appSecret,
-  config.zoom.domain
-);
+await ZoomUs.initialize({
+  clientKey: '...',
+  clientSecret: '...',
+})
 
 // Start Meeting
-await ZoomUs.startMeeting(
-  displayName,
-  meetingNo,
-  userId, // can be 'null'?
-  userType, // for pro user use 2
-  zoomAccessToken, // zak token
-  zoomToken // can be 'null'?
-
-  // NOTE: userId, userType, zoomToken should be taken from user hosting this meeting (not sure why it is required)
-  // But it works with putting only zoomAccessToken
-);
-
-// OR Join Meeting
-await ZoomUs.joinMeeting(
-  displayName,
-  meetingNo
-);
-
-// OR Join Meeting with password
-await ZoomUs.joinMeetingWithPassword(
-  displayName,
-  meetingNo,
-  'Enter password here'
-);
+await ZoomUs.startMeeting({
+  userName: 'Johny',
+  meetingNumber: '12345678',
+  userId: 'our-identifier',
+  zoomAccessToken: zak,
+  userType: 2, // optional
+})
+
+// Join Meeting
+await ZoomUs.joinMeeting({
+  userName: 'Johny',
+  meetingNumber: '12345678',
+})
+
+// Join Meeting with extra params
+await ZoomUs.joinMeeting({
+  userName: 'Johny',
+  meetingNumber: '12345678',
+  password: '1234',
+  participantID: 'our-unique-id',
```

Comment:
I've noticed an inconsistency with naming `userId` and `participantID`

Reply:
Yes. I also noticed it. It comes from SDK, so naming is ok. 
I think userId - means meeting initiator (host), participant - member

Reply:
In sdk they use `userID` and `participantID` so they are consistent.
I guess we should use `userID`, too, but that would be a breaking change of version 5 (but that's just been released so I guess that would be fine)

Reply:
Oh, yes. You right. I thought about field name. 
Maybe in future I'll add such update with support.